src/yang/ietf-bgp-neighbor.yang

```diff
@@ -190,6 +190,35 @@ submodule ietf-bgp-neighbor {
             "This leaf indicates whether the ability to support
              graceful-restart has been advertised to the peer";
         }
+        leaf local-forwarding-state-preserved {
+          type boolean;
+          config false;
+          description
+            "This leaf indicates whether the local router has
+             or would advertise the Forwarding State bit in its
+             Graceful Restart capability for this AFI-SAFI.";
+          reference
+            "RFC 4724: Graceful Restart Mechanism for BGP.";
+        }
+        leaf forwarding-state-preserved {
```

Comment:
Can you Rename this as a neighbor-forwarding-state-preserved or something that indicates its for the neighbor state?

Reply:
The neighbor portion is implied by the scope.  Do we really need it in there?

Reply:
Ack. nah. I see what you mean.